workspaces/simon-game/src/app/components/Box.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+type Props = { color?: string };
+
+export function Box(props: Props) {
+  const style = {
+    backgroundColor: props.color || "lightblue",
+    height: 100,
+    width: 100,
+  };
```

Comment:
Let's actually inline the style like before.

Reply:
Reverted to inlined style!